services/py-api/src/utils.py

```diff
@@ -43,3 +45,37 @@ async def stop(self) -> None:
             raise RuntimeError("The AsyncClient has not been initialized!")
 
         await self._async_client.aclose()
+
+class JwtUtility:
+    class UserData(TypedDict):
+        id: int
+        name: str
+        email: str
+        is_admin: bool
+        email_verified: bool
+        team_id: int
+        created_at: str
+        updated_at: str
+         
+
+    @staticmethod
+    def encode(data: UserData) -> str:
+            return encode(data,key=environ["SECRET_KEY"])
```

Comment:
Both of the methods the one from the jwt library and the one that you have defined on your own have the same name. This is ambiguous! please change the name of your own method to encode_data. Also, use the Participant form the participant model as a type for the data, as I explained in the previous comment.

Reply:
Fixed in be223527926be992bf6729e80ee043728b8fa1c5